unity-renderer/Assets/DCLPlugins/ECS7/Utils/Scene/Media.cs

```diff
@@ -0,0 +1,70 @@
+using DCL;
+using DCL.Models;
+using System;
+using System.Collections.Generic;
+
+/*
+* Scene utils file for media related stuff
+*/
+
+public static partial class UtilsScene
+{
+    // We are currently accepting all scene's required permissions.
+    // In the future we might want to replace this for user's accepted permissions.
+    public static bool HasRequiredPermission(IReadOnlyList<string> requiredPermissions, string permission)
+    {
+        for (int i = 0; i < requiredPermissions.Count; i++)
+        {
+            if (requiredPermissions[i] == permission)
+                return true;
+        }
+
+        return false;
+    }
+
+    public static bool IsUrlDomainAllowed(IReadOnlyList<string> allowedDomains, string url)
+    {
+        if (Uri.TryCreate(url, UriKind.Absolute, out Uri uri))
+        {
+            for (int i = 0; i < allowedDomains.Count; i++)
+            {
+                if (allowedDomains[i] == uri.Host)
+                    return true;
+            }
+        }
+
+        return false;
+    }
+
+    public static bool TrySetMediaUrl(string inputUrl, ContentProvider sceneContentProvider,
```

Comment:
Maybe I'm not understanding the method, but I would remove the `Set` from the name. It  looks like just getting the output url.

Reply:
Yeah you are right, don't know why did I put "Set" instead of "Get" 🤷‍♂️
changed here https://github.com/decentraland/unity-renderer/pull/4401/commits/e708091992e57c4333122cf3468e84a68ea879ea